x/ibc/light-clients/solomachine/types/client_state_test.go

```diff
@@ -85,10 +84,7 @@ func (suite *SoloMachineTestSuite) TestVerifyClientState() {
 	// create client for tendermint so we can use client state for verification
 	clientA, _ := suite.coordinator.SetupClients(suite.chainA, suite.chainB, ibctesting.Tendermint)
 	clientState := suite.chainA.GetClientState(clientA)
-
-	clientPrefixedPath := "clients/" + counterpartyClientIdentifier + "/" + host.ClientStatePath()
-	path, err := commitmenttypes.ApplyPrefix(prefix, clientPrefixedPath)
-	suite.Require().NoError(err)
+	path := suite.solomachine.GetClientStatePath(counterpartyClientIdentifier)
```

Comment:
I think ideally we abstract this in `host` for all path type's so the users doesn't need to call `ApplyPrefix`. I only abstracted this in solo machine because I started to write dup code and then figured I might as well do it for all path types